sycl/cmake/modules/UnifiedRuntimeTag.cmake

```diff
@@ -1,7 +1,7 @@
-# commit 9937d029c7fdcbf101e89f8515f640c145e059c5
-# Merge: 9ac6d5d9 10b0e101
-# Author: Callum Fare <callum@codeplay.com>
-# Date:   Wed Nov 20 14:49:17 2024 +0000
-#     Merge pull request #2258 from aarongreig/aaron/tryUseExtensionSubgroupInfo
-#     Use extension version of clGetKernelSubGroupInfo when necessary.
-set(UNIFIED_RUNTIME_TAG 9937d029c7fdcbf101e89f8515f640c145e059c5)
+# commit 20e501a036e5e16368eb5c8f1e92289efb3ea735
+# Merge: dcf35a10 ca878c58
+# Author: Piotr Balcer <piotr.balcer@intel.com>
+# Date:   Thu Nov 21 10:55:59 2024 +0100
+#   Merge pull request #1975 from Bensuo/fabio/immediate_append_exp
+#   [Command-Buffers] Implement new command-list enqueue path
+set(UNIFIED_RUNTIME_TAG 20e501a036e5e16368eb5c8f1e92289efb3ea735)
```

Comment:
https://github.com/intel/llvm/pull/15843 has merged, which bumps UR past this tag - so I think we can remove touching the tag from this PR.

Reply:
I've merged in `sycl` to remove this